src/Main.hs

```diff
@@ -85,6 +88,8 @@ data Opts = Opts
   { optsToken :: AuthToken
   , optsOffline :: Bool
   , optsIgnoreCase :: Bool
+  , optsSearchAll :: Bool
+  , optsShowDate :: Bool
   , optsBy :: Maybe String
   , optsOrganisation :: ParamName Organisation
   , optsFlow :: ParamName Flow
```

Comment:
This could be modified so `optsFlow :: Maybe (ParamName Flow)`, so `Nothing` indicates to search all flows. Then the parser could be modified to either take `--all` or require flow name.

Currently with `--all` you'd need to specify dummy flow name, don;t you?